packages/block-library/src/spacer/edit.native.js

```diff
@@ -0,0 +1,62 @@
+
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
You don't need to explicitly import `React`. It's handled with the Babel preset.